draft-ietf-rtcweb-jsep.xml

```diff
@@ -1901,8 +1970,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>A direction attribute which is the same as that of the
               associated transceiver.</t>
 
-              <t>For each supported codec, "a=rtpmap" and "a=fmtp"
-              lines, as specified in
+              <t>For each supported codec that has not been excluded by
+              the codec preferences of the associated transceiver,
+              "a=rtpmap" and "a=fmtp" lines, as specified in
```

Comment:
I think you would be better if you phrased this as
"For each codec that is available for this transceiver" rather than "supported filtered by excluded"